common/changes/@uifabric/variants/bodyDividerSlot5.0_2018-09-04-18-34.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/variants",
+      "comment": "Change bodyDivider value from neutralTertiaryAlt back to neutralLight",
+      "type": "minor"
```

Comment:
I don't see a change for variants in this PR...

Reply:
Removed log!